src/platform/site-wide/cta-widget/index.js

```diff
@@ -430,40 +412,54 @@ export class CallToActionWidget extends React.Component {
       );
     }
 
-    const {
-      heading,
-      alertText,
-      buttonText,
-      buttonHandler,
-      status = 'info',
-    } = this.getContent();
-
-    const alertProps = {
-      headline: heading,
-      content: (
-        <div className="usa-alert-text">
-          {alertText}
-          {buttonText && (
-            <button className="usa-button-primary" onClick={buttonHandler}>
-              {buttonText}
-            </button>
-          )}
-        </div>
-      ),
-      status,
-    };
+    const content = this.getContent();
+
+    if (content) {
+      const {
+        heading,
+        alertText,
+        buttonText,
+        buttonHandler,
+        status = 'info',
+      } = content;
+
+      const alertProps = {
+        headline: heading,
+        content: (
+          <div className="usa-alert-text">
+            {alertText}
+            {buttonText && (
+              <button className="usa-button-primary" onClick={buttonHandler}>
+                {buttonText}
+              </button>
+            )}
+          </div>
+        ),
+        status,
+      };
```

Comment:
I kinda feel like this could be abstracted into its own component, `<CTAAlert/>`?